### PR TITLE
Improve github action

### PR DESCRIPTION
**Pull request by ferhatelmas**
* whitespace (trim and indentation)
* step naming
* drop `id` from setup

Related to the comment: https://github.com/kubernetes-sigs/krew/pull/491#discussion_r374563885

Own initiative, feel free to close abruptly.

### CODE REVIEW

**Comment by ahmetb** [comment]
Can we not do the indentation? I find unnecessary yaml indents superfluous. :)

**Comment by ferhatelmas** [comment]
> Can we not do the indentation? I find unnecessary yaml indents superfluous. :)

As you wish.

In general, I setup [lint](https://github.com/adrienverge/yamllint) in my environment.

**Comment by ahmetb** [comment]
YAML indentation is not done in Kubernetes ecosystem, in general so avoiding would be the best.

**Comment by ferhatelmas** [comment]
> YAML indentation is not done in Kubernetes ecosystem, in general so avoiding would be the best.

Better to be consistent. Reverted indentation but kept the trimming.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,47 +23,47 @@ jobs:
...
     - name: Install dependencies
       run: go mod download
-    
+
     - name: Install gox
       run: hack/install-gox.sh
-      
-    - name: Verify code quality
+
+    - name: Ensure go modul is tidy
```
Comment (corneliusweig) [inline .github/workflows/ci.yml]: There's a typo here.

Comment (ahmetb) [inline .github/workflows/ci.yml]: Ensure go.mod is already tidied

**Comment by ahmetb** [comment]
nits but /lgtm

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**